routes/api.php

```diff
@@ -200,20 +200,25 @@
 
     /*--- Fields API ---*/
 
+    Route::resource('fields', 'CustomFieldsController', [
+        'names' => [
+            'index' => 'api.fields.index',
```

Comment:
Since we use the `customfields` route name elsewhere, can you change these to reflect that? `api.customfields.show`, etc?

Reply:
I just extended the existing route, so there should be no conflicts. There are no differences between the current and my implementation